### PR TITLE
Upgrade alpine to v3.13

### PR DESCRIPTION
**Pull request by ChrisSamo632**
* Upgrade to alpine 3.13
* Update `.drone.yml` to use latest ACP docker-in-docker images and approach for builds
* Removed Anchore scan step from Drone build as it wasn't actually being used (just spun up as a service that was then ignored)

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -15,23 +10,30 @@ platform:
...
   # wait for docker service to be up before running docker build
-  - n=0; while [ "$n" -lt 60 ] && [ ! -e /var/run/docker.sock ]; do n=$(( n + 1 )); sleep 1; done
+  - n=0; while [ "$n" -lt 60 ] && ! docker stats --no-stream; do n=$(( n + 1 )); sleep 1; done
+  - if ! docker stats --no-stream; then echo "Docker appears not to be running"; exit 1; fi
   - docker build -t alpine:$${DRONE_COMMIT_SHA} . --no-cache
-  volumes:
-  - name: dockersock
-    path: /var/run
+  environment:
+    DOCKER_HOST:
```
Comment (KashifSaadat) [inline .drone.yml]: You can remove this env var when using the ACP dind image

Comment (ChrisSamo632) [reply .drone.yml]: Good catch - copied this from pipeline using `quay.digital.homeoffice.gov.uk/ukhomeofficedigital/dind-awscli:19.03.12-dind-1.18.55` as the Drone step's base image, which *does* need the extra env var.

Comment (KashifSaadat) [reply .drone.yml]: Oh I wasn't aware of that, must have been an initial image before it was simplified.

**Review by KashifSaadat — APPROVED** [review]
Looks good, thanks Chris! 👍

**Pull request merged**